packages/discord.js/typings/index.test-d.ts

```diff
@@ -566,9 +566,11 @@ client.on('guildCreate', async g => {
     expectType<ThreadMember>(fetchedMember);
     expectType<ThreadMember>(fetchedMember2);
     const fetchedMemberCol = await channel.members.fetch({ cache: true });
+    const fetchedMemberCol2 = await channel.members.fetch();
     expectType<Collection<Snowflake, ThreadMember>>(fetchedMemberCol);
+    expectType<Collection<Snowflake, ThreadMember>>(fetchedMemberCol2);
     // @ts-expect-error The `force` option cannot be used alongside fetching all thread members.
-    const fetchedMemberCol2 = await channel.members.fetch({ cache: true, force: false });
+    const fetchedMemberCol3 = await channel.members.fetch({ cache: true, force: false });
```

Comment:
We should either use meaningful names, or use blocks to declare variables without name conflicts. An example of the latter would be:

```ts
{
	const result = await channel.members.fetch();
	expectType<Collection<Snowflake, ThreadMember>>(result);
}
{
	const result = await channel.members.fetch({ cache: true });
	expectType<Collection<Snowflake, ThreadMember>>(result);
}
```

Also makes the separation between type tests clearer.

Reply:
To be honest, I want to get rid of this entire `if` block. The `ThreadMemberManager` is only being tested here, but the other managers are on lines 940 to 1113. What would you say about just moving it there alongside other managers?